sklearn/base.py

```diff
@@ -292,6 +293,24 @@ def __repr__(self):
         return '%s(%s)' % (class_name, _pprint(self.get_params(deep=False),
                                                offset=len(class_name),),)
 
+    def __getstate__(self):
+        if type(self).__module__.startswith('sklearn.'):
+            return dict(self.__dict__.items(), __version__=__version__)
+        else:
+            return dict(self.__dict__.items())
```

Comment:
Why hack? This adds the information to the file when the estimator is stored.
What would be the benefit of a ``version_`` attribute outside of serialization?

The ``__setstate__`` would still need to be changed the same way, right? (only without a pop)